cli/src/commands/tui/app.rs

```diff
@@ -100,6 +100,10 @@ pub struct State {
     pub builds_error: Option<String>,
     pub deployments_error: Option<String>,
 
+    // Auth
+    pub is_gcloud_authenticated: Option<bool>,
+    pub is_okta_authenticated: Option<bool>,
```

Comment:
Do they need to be `Option`?

Reply:
Yes, reason being is that it has 3 states, not verified, verifying and failed
So i can see this can be accomplished by having an Option or 3 enums. 

^ above states is used to display the UI for the user